### ISSUE TITLE
Style the Package Version Select

### ISSUE THREAD

**Issue by sabine** [ocaml/ocaml.org#1612]
This is an Outreachy Medium difficulty issue. You must have completed a simple issue (marked with label "outreachy") first, to be eligible for being assigned to this one.

This issue is for the Dark Mode project. Only request to be assigned this issue if you are applying for the Outreachy Dark Mode project.

# Task

On the package documentation pages, there is a select element to switch between different versions of the same package. This should be replaced by a dropdown like this:

|before|after|
|-|-|
|![Screenshot from 2023-10-11 11-24-56](https://github.com/ocaml/ocaml.org/assets/6594573/4ee6b9dd-2456-4e6e-a48c-d78debf271d7)|![Screenshot from 2023-10-11 11-24-45](https://github.com/ocaml/ocaml.org/assets/6594573/737cd7e4-13c6-43c5-ab16-352fdd02113b)|

[Here is a Figma Document with the styles](https://www.figma.com/file/J0w4b9exaHhVpvabKoGqTn/Design-Github-Issues?type=design&node-id=47%3A274&mode=design&t=q6e3n0Ul6MatGJqY-1).

Constraints:
* Dropdown implementation must be accessible (WCAG).

How to solve this issue:
1. Determine a reasonable, accessible implementation for the version switcher dropdown. AlpineJS may be used.
2. Modify `package_breadcrumbs.eml`.

**Comment by FatumaA** [comment]
Please assign this issue as all first issues with outreachy are already assigned
@sabine @SaySayo

**Comment by SaySayo** [comment]
Have you completed a beginner issue yet? @FatumaA

**Comment by mohdaquib171** [comment]
Please assign this to me @SaySayo, i already completed simple issue, and also merged pull request

**Comment by FatumaA** [comment]
No. All beginner issues labelled "outreachy" have already been assigned

**Comment by SaySayo** [comment]
You cannot a medium issue without first completing a beginner issue

**Comment by SaySayo** [comment]
Also @mohdaquib171, you have already been assigned a different issue

**Comment by FatumaA** [comment]
Ok, will there be more beginner issues? I have seen some issues labelled ui without the outreachy label

**Comment by SaySayo** [comment]
I cannot say for certain, the dark mode project closes to new contributors tomorrow and only those that currently have a recorded contributions will be eligible to apply to it as there are already lots of applicants.

**Comment by FatumaA** [comment]
Can i tag you on some of the ui issues for evaluation of outreachy fit?

**Comment by SaySayo** [comment]
yes you can

**Comment by shyusu4** [comment]
I want to catch up and apply for the dark mode project asap, but I'm still working on my first issue. (not sure if the first issue I got assigned is suitable for gui or dark mode)
Will I still be able to be assigned this issue even though I'm still working on my first issue? cc @sabine @SaySayo

**Comment by oyenuga17** [comment]
Hi @sabine, @SaySayo please I'd like to be assigned to this issue if possible. I'm free now and open to more issues 🙏

**Comment by SaySayo** [comment]
Hi @oyenuga17, your enthusiasm is very much welcome, But as there is currently a scarcity of outreachy issues to fix, we're trying to make it go round so that all interested applicants have the opportunity to make contributions.

**Comment by SaySayo** [comment]
Hello @shyusu4, what first issue are you referring to please? Also do you have the project setup locally? I have not seen a screenshot from you.

**Comment by oyenuga17** [comment]
> Hi @oyenuga17, your enthusiasm is very much welcome, But as there is currently a scarcity of outreachy issues to fix, we're trying to make it go round so that all interested applicants have the opportunity to make contributions.

alright, thank you @SaySayo, I'll be waiting in line 😊 🙏

**Comment by mohdaquib171** [comment]
I've just created a pull request for the earlier issue #1617  and would like to contribute further. If possible, could you please consider assigning this issue to me as well?

> Also @mohdaquib171, you have already been assigned a different issue

**Comment by shyusu4** [comment]
> Hello @shyusu4, what first issue are you referring to please? Also do you have the project setup locally? I have not seen a screenshot from you.

@SaySayo I've been assigned issue https://github.com/ocaml/ocaml.org/issues/1594. And yes, I've set up the project locally.

![Screenshot (345)](https://github.com/ocaml/ocaml.org/assets/106587342/14ef0fdf-9513-47af-b0f9-d84c7ed482bf)

**Comment by SaySayo** [comment]
Hi @mohdaquib171, your enthusiasm is very much welcome, But as there is currently a scarcity of outreachy issues to fix, we're trying to make it go round so that all interested applicants have the opportunity to make contributions.

**Comment by SaySayo** [comment]
@shyusu4, you cannot be assigned a new issue until you have completed your first PR

**Comment by ademolaomosanya** [comment]
i would love to work on this project just completed my first issue
here is the PR: #1616

**Comment by SaySayo** [comment]
You shall be assigned this issue, @ademolaomosanya. Let me know if you need any help. cc @sabine

**Comment by IdaraNabuk** [comment]
@SaySayo I would love to work on this issue. I just completed my first tasks and here's the PR: https://github.com/ocaml/ocaml.org/pull/1620

**Comment by ademolaomosanya** [comment]
> You shall be assigned this issue, @ademolaomosanya. Let me know if you need any help. cc @sabine

okay will start working on it right away

**Comment by Aquilaafuadajo** [comment]
Hi @SaySayo @sabine I see this issue has not been worked on yet, please assign to me I'd love to give it a try 😊

**Comment by ademolaomosanya** [comment]
@Aquilaafuadajo i'm open to collaboration with you on this task have not figured it out your help will be appreciated 
@sabine @SaySayo

**Comment by SaySayo** [comment]
@ademolaomosanya you can go ahead and open a PR to this issue. @Aquilaafuadajo is working on another issue.

**Comment by Aquilaafuadajo** [comment]
Okay @SaySayo, Thanks for the update.
